### PR TITLE
[docs] Build release notes to documentation

### PR DESCRIPTION
**Pull request by bmorelli25**
For https://github.com/elastic/apm-agent-rum-js/issues/511.

This PR builds release notes to the documentation by replacing `CHANGELOG.md` with `CHANGELOG.asciidoc`.

This PR is currently blocked by https://github.com/elastic/apm-agent-rum-js/pull/512 & elastic/docs#1518. As a result, the docs/ci will not pass.

## Because who doesn't like screenshots?
Screen Shot 2019-12-04 at 12 27 50 PM
Screen Shot 2019-12-04 at 12 27 40 PM

### CODE REVIEW

**Comment by bmorelli25** [comment]
It sounds like the process would remain the same, just instead of copying to `changelog.md`, you'd copy to `changelog.asciidoc` (and change the format to match).

**Comment by bmorelli25** [comment]
Sorry, I didn't realize there were separate changelogs for each package. What if we change all off the changelogs to asciidoc files and then include them in the top level changelog file. Then there would be no need to manually copy/paste before each release

**Comment by vigneshshanmugam** [comment]
Not sure if that would work, since those `changelogs` are auto generated and it is based on how we annotate the commit. So I guess copying those changelog to master changelog.asciidoc sounds good to me.

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -0,0 +1,501 @@
+ifdef::env-github[]
+NOTE: Release notes are best read in our documentation at
+https://www.elastic.co/guide/en/apm/agent/go/current/release-notes.html[elastic.co]
```
Comment (jahtalab) [inline CHANGELOG.asciidoc]: This is referencing the go agent, it should be `https://www.elastic.co/guide/en/apm/agent/rum-js/current/release-notes.html`

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -0,0 +1,501 @@
...
+[float]
+===== Breaking changes
+
+[float]
+===== Features
+* Cool new feature: {pull}2526[#2526] {commit}d4f181f[#d4f181f]
+
+[float]
+===== Bug fixes
+////
```
Comment (jahtalab) [inline CHANGELOG.asciidoc]: This template is great, thanks 👍

File: CHANGELOG.asciidoc [hunk 3]
```diff
@@ -0,0 +1,501 @@
...
+==== x.x.x (YYYY/MM/DD)
+
+[float]
+===== Breaking changes
+
+[float]
+===== Features
+* Cool new feature: {pull}2526[#2526] {commit}d4f181f[#d4f181f]
+
+[float]
```
Comment (vigneshshanmugam) [inline CHANGELOG.asciidoc]: have to include `Performance Improvements` to the list. example - https://github.com/elastic/apm-agent-rum-js/blob/master/CHANGELOG.md#performance-improvements

**Comment by bmorelli25** [comment]
@jahtalab @vigneshshanmugam -- If the ci failure looks ok, please merge. Sorry for the delay on this!

**Pull request merged**